webauthn/idlharness.https.any.js

```diff
@@ -0,0 +1,60 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+// META: script=helpers.js
+
+// https://w3c.github.io/webauthn/
+
+'use strict';
+
+promise_test(() => {
+  const execute_test = () => idl_test(
+    ['webauthn'],
+    ['credential-management'],
+    idlArray => {
+      idlArray.add_objects({
+        WebAuthentication: ['navigator.authentication'],
+        PublicKeyCredential: ['cred'],
+        AuthenticatorAssertionResponse: ['assertion']
```

Comment:
@jcjones @apowers313 - I'm not sure how to check whether I've set up these tests correctly.

In short, these two fields `self.cred` and `self.assertion` are supposed to be a `PublicKeyCredential` and a `AuthenticatorAssertionResponse` respectively. Is that what you would expect to result from the code below? I can see the prompt in FF61, but how do I add a security token to Firefox?